tools/rna_tools/mlocarna/macros.xml

```diff
@@ -0,0 +1,102 @@
+<macros>
+    <xml name="requirements">
+        <requirements>
+            <requirement type="package" version="1.8.11">locarna</requirement>
+        </requirements>
+    </xml>
+    <xml name="stdio">
+        <stdio>
+            <exit_code range="1:" />
+        </stdio>
+    </xml>
+
+    <xml name="version">
+        <version_command>
+            <![CDATA[
+                     mlocarna --version
+            ]]>
+        </version_command>
+    </xml>
+    
+    <xml name="common_scoring_parameters">
+        <param name="struct_weight" label="Structure Weight" type="integer" 
+               value="200" help="--struct-weight" />
+        <param name="indel_opening" label="Indel Opening Score" type="integer"
+               value="-500" help="--indel-opening" />
+        <param name="indel" label="Indel score" type="integer" value="-350" help="--indel" />
+        <param name="tau" type="integer" value="50" 
+               label="Sequence contribution at structure match in percent"
+               help="--tau factor"/> 
+
+        <conditional name="sequence_score">
+            <param name="sequence_score_selector" type="select" label="Type of sequence score contribution">
+                <option value="ribofit">Use ribofit</option>
+                <option value="ribosum">Use RIBOSUM85_60</option>
+                <option value="match">Simple match/mismatch costs</option>
+            </param>
+            <when value="ribofit" />
+            <when value="ribosum" />
+            <when value="match">
+                <param name="match" type="integer" value="50" label="Match score" />
+                <param name="mismatch" type="integer" value="0" label="Mismatch score" />
+            </when>
+        </conditional>
+    </xml>
+
+    <xml name="common_folding_parameters">
+        <param name="plfold_span" type="integer" value="150" 
+               label="Maximum basepair span (local folding); -1 for global folding"
+               help="--plfold-span=span"/>
+        
+        <param name="plfold_winsize" type="integer" value="300" 
+               label="Window size for local folding"
+               help="--plfold-winsize=winsize"/>
+        
+        <param name="rnafold_temperature" type="float" value="37.0"
+               label="Temperature for RNAfold (RNAfold's -T option)"
+               help="--rnafold-temperature=temp" />
+    </xml>
+
+
+
+    <xml name="common_heuristic_parameters">
+        <param  name="min_prob" type="float" value="0.0005" label="Minimal probability" help="-p cutoff" /> 
```

Comment:
Do you have some `min` `max` recommendations? Including min, max will render a nice slider in the Galaxy UI.